Code/Tools/RemoteConsole/Core/RemoteConsoleCore.cpp

```diff
@@ -232,12 +253,10 @@ void SRemoteServer::Run()
             continue;
         }
 
-        m_lock.Lock();
-        m_stopEvent.Reset();
+        AZStd::lock_guard<AZStd::recursive_mutex> lock(m_mutex);
```

Comment:
Can be replaced with

```suggestion
        AZStd::scoped_lock lock(m_mutex);
```